recipe/conda_build_config.yaml

```diff
@@ -56,6 +88,14 @@ zip_keys:
     - vc                        # [win]
     - c_compiler                # [win]
     - cxx_compiler              # [win]
+  -
+    - c_compiler                # [linux or osx]
+    - cxx_compiler              # [linux or osx]
+    - fortran_compiler          # [linux or osx
+    - channel_sources           # [linux or osx]
+    - channel_targets           # [linux or osx]
+    - docker_image              # [linux]
+    - build_number_increment    # [linux or osx]
```

Comment:
It's true that `build_number_increment` is not needed for Windows, but it's also not the nicest thing to have diverging build numbers depending on the platform..

Reply:
fair, we can probably just have that be a static 1000 in that case